### PR TITLE
Add logo to docs

### PR DESCRIPTION
**Pull request by nathanleclaire**
cc @ehazlett I think it would look nice.

Signed-off-by: Nathan LeClaire <nathan.leclaire@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Docker Machine
 
+![](/docs/img/logo.png)
```
Comment (thaJeztah) [inline README.md]: Perhaps add an "alt-text" here, e.g. `![Docker Machine Logo](/docs/img/logo]`

**Comment by thaJeztah** [comment]
I absolutely love the logos, so yeah! Couple of things that need to be checked/verified;

- Are the logos available in a vector format (SVG?), if so, those may be preferred (at the chance it's not supported on older browsers)
- Given that all the the docs files of the various projects are merged, I *think* the files should have unique names, so `logo.png` should be something like `machine-logo.png` (cc @moxiegirl to confirm I'm right here)
- Should we add credits for the logo's / artwork somewhere?

**Comment by moxiegirl** [comment]
@thaJeztah I've heard different things about image formats.  We probably need to define a policy and follow it.  https://quickleft.com/blog/choosing-svg-jpeg-png/ might be a start.  For this, I wouldn't hold the PR up on the image format.

The project README.md file is not part of the docs.docker.com source set.  @thaJeztah  is correct, we are trying to standardize on  `images` as a directory.  In production, these appear in `project/images`.  Result, is you could leave it as `logo.png` as the risk for name collision is pretty low.  @thaJeztah I'm finally getting to the task of writing up docs workflow etc this week, so everyone has a better understanding.

Docker owns the rights to the images that Laurel creates for us.  She is credited as their creator all over.  So, I don't think we need to credit her specially here.

**Comment by moxiegirl** [comment]
LGTM

**Comment by thaJeztah** [comment]
LGTM

Thanks for the extensive explanation; I wanted to be sure we were doing the right thing here (e.g. credits are easily overlooked)

Thanks @nathanleclaire! (sorry, can't press merge for you as I don't have permissions here)

**Comment by vieux** [comment]
:+1:

**Comment by hairyhenderson** [comment]
Nice. :+1:

**Comment by thaJeztah** [comment]
ping @moxiegirl @ehazlett for the merge :-)

**Comment by ehazlett** [comment]
Yay :+1:

**Pull request merged**